samples/qa-a11y/src/views/SliderButton.js

```diff
@@ -0,0 +1,22 @@
+import SliderButton from '@enact/agate/SliderButton';
+import React from 'react';
+
+import Section from '../components/Section';
+
+import appCss from '../App/App.module.less';
+
+const SliderButtonView = () => (
+	<>
+		<Section title="Default">
+			<SliderButton alt="Normal" onChange={Function.prototype}>{['A', 'B', 'C']}</SliderButton>
```

Comment:
Is `onChange` prop required? I saw your mention in the description but I think we should fix the error instead of this.

Reply:
onChange={Function.prototype} was removed.